style-guide/template-style-guide.md

```diff
@@ -0,0 +1,132 @@
+<!-- Copy this Template. -->
+<!-- Change the name of your project, such as "Good Docs style guide". -->
+# Style guide template
+
+<!-- Before using this template, please read the accompanying About the
+Style Guide Template documentation. -->
+
+## Introduction
+
+Welcome to our project! This style guide is intended for use by project
+contributors, not necessarily end-users. It provides general guidance to anyone
+who contributes to the project’s documentation.
+
+## Our Preferred Style Guide
+
+We have adopted the
+[your preferred style guide, such as the Google Developer's Style Guide](https://developers.google.com/style)
+for {Your Project} documentation. For a quick summary, see the
+[Google style guide highlights](https://developers.google.com/style/highlights).
+The rest of this document describes our project-specific customizations to
+{Google's or your preferred} guide.
+
+Our project uses {your choice: standard American spelling or standard British
+spelling, etc.} and our preferred dictionary is the
+{[American Heritage Dictionary](https://ahdictionary.com/) or your preferred
+dictionary}.
+
+When writing documentation for our project, align with the default guide’s
+voice and tone.
+
+## Glossary of Preferred Terms
+
+<!-- This first paragraph is optional or you could include in the word list. -->
+
+This {Your Project Name} is represented as {example}. It is {always/never}
+capitalized.
+
+The table provides guidelines about the terms you should and should not use for
+consistency, listed in alphabetical order:
+
+Preferred Term  | Avoid These Terms  |  Explanation
+--------------- | -----------------  |  -----------
+for example     | e.g.               |  Avoid non-English words
+people with disabilities  | <ul><li>the disabled</li><li>disabled people</li><li>people with handicaps</li><li>the handicapped</li></ul>  |  Use inclusive and bias-free language. See [Accessible Writing](#accessible-writing
+that is         |  i.e.              |  Avoid non-English words
+
+## Topic Types and Templates
+
+This project recommends using the following templates from the
+[Good Docs project](https://github.com/thegooddocsproject/templates):
+
+- API Overview
+- API Quickstart
+- API Reference
+- Discussion Article
+- How To Guide
+- Logging Article
+- Reference Article
+- Tutorial
+
+## General Writing Tips
+
+<!-- This section is optional -->
+
+For some general tips about improving writing see:
+
+- [Write the Docs - Style Guides](https://www.writethedocs.org/guide/writing/style-guides/#writing-style)
+- [18F Content Guide](https://content-guide.18f.gov/)
+
+## Accessible Writing
+
+Documentation should be written in a way that supports people with disabilities
+and users with various input methods and devices. Improving accessibility also
+helps make documentation clearer and more useful for everyone.
+
+For resources on making your writing more accessible, see:
```

Comment:
I've made a few suggestions to use the linked resources' own titles and capitalization. I think diverging on this sort of thing can lead to needless confusion (e.g., the Microsoft Writing Style Guide is the successor to the Manual of Style, which is no longer published). Plus, it's easier to tell I've landed on the right page.

Because there were a few of them in one place, I flagged them here, but some of these get used more than once and I didn't trust I'd get them all. You'll want to check throughout for each of these.

Reply:
I realized I hadn't entirely fixed this issue that you raised, so I've gone through and cleaned up all the references to Google developer documentation style guide and Microsoft Writing Style Guide now. I checked any references to other sources should now be correct with my latest commit.

Reply:
Oh, good catch. I forgot to recheck this myself. Thank you!